### PR TITLE
Added task cleanup. Updated README

### PR DESCRIPTION
**Pull request by indvamshi**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RWA-219

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,72 +39,69 @@ To build the project execute the following command:
...
+        ```
+            yarn upload-wa
+        ```
+- To run application, from IDE or command line
+   ```
+     ./grdlew bootRun
+   ```
+- In order to test if the application is up, you can call its health endpoint:
+   ```
+     http://http://ccd-data-store-api/
```
Comment (alexbaeza) [inline README.md:71]:
```suggestion
     http://http://ccd-data-store-api/health
```

**Review by alexbaeza — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -39,72 +39,68 @@ To build the project execute the following command:
...
+       ```
+         ./grdlew bootRun
+       ```
+    - Run ia-case-notifications
+       ```
+         ./grdlew bootRun
+       ```
+    - Run ia-case-documents-api
+       ```
+         ./grdlew bootRun
```
Comment (alexbaeza) [inline README.md:91]:
```suggestion
         ./gradlew bootRun
       ```
    - Run ia-case-notifications
       ```
         ./gradlew bootRun
       ```
    - Run ia-case-documents-api
       ```
         ./gradlew bootRun
```

File: README.md [hunk 3]
```diff
@@ -39,72 +39,68 @@ To build the project execute the following command:
...
+        ```
+            yarn setup
+        ```
+    - Upload case definitions
+        ```
+            yarn upload-wa
+        ```
+- To run application, from IDE or command line
+   ```
+     ./grdlew bootRun
```
Comment (alexbaeza) [inline README.md:67]:
```suggestion
     ./gradlew bootRun
```

**Review by alexbaeza — APPROVED** [review]

**Review by iguannago — APPROVED** [review]

File: src/functionalTest/java/uk/gov/hmcts/reform/wataskconfigurationapi/BaseFunctionalTest.java [hunk 4]
```diff
@@ -31,4 +33,14 @@ public void setUp() throws Exception {
...
 
+    public void cleanUp(String taskId, String token) {
+        given()
+            .header(SERVICE_AUTHORIZATION, token)
+            .contentType(APPLICATION_JSON_VALUE)
+            .baseUri(camundaUrl)
+            .basePath("/task/" + taskId + "/complete")
+            .when()
+            .post();
+    }
```
Comment (BenNzewi) [inline src/functionalTest/java/uk/gov/hmcts/reform/wataskconfigurationapi/BaseFunctionalTest.java:44]: Can we check the state of the completed task? I.e. check that the task no longer exists as part of completing and clean up.

**Review by alexbaeza — APPROVED** [review]

**Review by iguannago — DISMISSED** [review]

**Review by iguannago — APPROVED** [review]

**Pull request merged**